### PR TITLE
Docs: Add documentation for java8_home property

### PR DESCRIPTION
**Pull request by rschlussel**
Testing: make docs

The only bits that are actually changed are lines 30-52 in the new file/ 20-30 in the old.
The rest of the changes are just where I broke up the really long lines into several lines so that it's easier to read in an editor.  This does not affect the way the docs look when compiled

@ebd2 @akshatnair

### CODE REVIEW

File: docs/installation/presto-admin-configuration.rst [hunk 1]
```diff
@@ -27,7 +41,15 @@ All of the properties are optional. If you include a blank ``config.json`` file,
  "workers": ["localhost"]
  }
 
-The above configuration is for a single-node installation of Presto on the same node that ``presto-admin`` is installed on. You can also specify some but not all of the properties. For example, for a 6 node cluster with default username and port, a sample ``config.json`` would be:
+Note that ``java8_home`` is not set by default.  It only needs to be set if
+Java 8 is in a non-standard location on the Presto nodes.  The property is used
+to tell the presto rpm where to find Java 8.
+
+
```
Comment (akshatnair) [inline docs/installation/presto-admin-configuration.rst]: is this extra new line needed?

File: docs/installation/presto-admin-configuration.rst [hunk 2]
```diff
@@ -3,11 +3,23 @@
...
 Presto-Admin Configuration
 ==========================
-A Presto cluster consists of a coordinator node and one or more workers nodes. A coordinator and worker may be located on the same node, meaning that you can have a single-node installation of Presto, but having a dedicated node for the coordinator is recommended for better performance, especially on larger clusters.
-
-In order to use ``presto-admin`` to manage software on a cluster of nodes, you must specify a configuration for ``presto-admin``. This configuration indicates the nodes on which to install as well as other credentials.
-
-To set up a configuration, create a file ``/etc/opt/prestoadmin/config.json`` with the content below. Replace the variables denoted with brackets <> with actual values enclosed in quotations. The user specified by ``username`` must have sudo access on all of Presto nodes, and ``presto-admin`` also must be able to login to all of the nodes via SSH as that user (see :ref:`ssh-configuration-label` for details on how to set that up). The file should be owned by root with R/W permissions (i.e. 622).
+A Presto cluster consists of a coordinator node and one or more workers nodes.
```
Comment (akshatnair) [inline docs/installation/presto-admin-configuration.rst]: This formatting change is needed to adhere to some guideline? or just your settings?

**Comment by akshatnair** [comment]
lgtm mod small comment

File: docs/installation/presto-admin-configuration.rst (hunk 1 above)
Comment (rschlussel) [inline docs/installation/presto-admin-configuration.rst]: Yes.  If you have only one new line it basically gets ignored and interpreted as a space.  If you actually want two paragraphs, you need two newlines.

File: docs/installation/presto-admin-configuration.rst (hunk 2 above)
Comment (rschlussel) [inline docs/installation/presto-admin-configuration.rst]: The format change isn't needed for anything.  It just makes it easier to read.  Single newlines in markdown are basically equivalent to spaces.  All of the lines will together still be considered one paragraph for html purposes.  Whoever initially wrote this doc didn't realize that and so put everything that was supposed to be one paragraph on a single super long line.

File: docs/installation/presto-admin-configuration.rst (hunk 1 above)
Comment (rschlussel) [inline docs/installation/presto-admin-configuration.rst]: oh i see what you mean.  sorry.  I'll remove it.

**Pull request merged**